app/scripts/views/settings.js

```diff
@@ -206,6 +206,9 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
         })
         .fin(function () {
           self.logViewEvent('signout.success');
+          // Unset uid otherwise it will henceforth be impossible
+          // to sign in to different accounts inside this tab.
+          self.relier.unset('uid');
           self.user.clearSignedInAccount();
```

Comment:
Ah, my comment was hidden when you pushed. We should clear the `email` and `preVerifyToken` fields too. I'm not sure they'll cause any damage, but we might as well ditch any user-specific state while we are digging around in here.